pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -705,26 +705,24 @@ func (cb *ClusterBuilder) buildDefaultPassthroughCluster() *cluster.Cluster {
 			v3.HttpProtocolOptionsType: passthroughHttpProtocolOptions,
 		},
 	}
-	cb.applyConnectionPool(cb.req.Push.Mesh, NewMutableCluster(cluster), &networking.ConnectionPoolSettings{})
```

Comment:
Why this is removed?

Reply:
my mistake. I thought i need to revert my change